source/Projectile.js

```diff
@@ -0,0 +1,23 @@
+var Pixi = require("pixi.js")
+var Reference = require("Reference.js")
+
+export default class Projectile extends Pixi.Sprite {
+    constructor(x, y) {
+        var projectileTex =
+            Pixi.Texture.fromImage(require("./images/projectile.png"))
+        super(projectileTex)
+        this.x = x
+        this.y = y
+        this.speed = 80
+        console.log("Creating projectile")
```

Comment:
Aren't `console.log` great for debugging? So so handy! Eventually, though, we don't want to leave too many of these, or it'll make it too difficult which `console.log` we're looking for.